schema.graphql

```diff
@@ -33,19 +34,15 @@ type Transaction @entity {
   memo: String
   status: TxStatus!
   log: String
-  #  timeline: BigInt! @index
   timeoutHeight: BigInt @index
-  # NB: only the first signer!
   signerAddress: String @index
   messages: [Message] @derivedFrom(field: "transaction")
 }
 
 type Message @entity {
   id: ID!
-  #  typeUrl: String! @index
   typeUrl: String @index
```

Comment:
It seems to me like every message should have a type URL. Do we know why `typeUrl` is not required (i.e. not `String!`)? :thinking:

Reply:
I'll set it required